examples/tutorial-standalone/harness.tf

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  kek_keyring                 = "kek_keyring_${random_id.suffix.hex}"
+  kek_key_name                = "kek_key_${random_id.suffix.hex}"
+  key_rotation_period_seconds = "2592000s" #30 days
+  secret_name                 = "wrapped_key"
+  wrapped_key_secret_data     = chomp(data.google_secret_manager_secret_version.wrapped_key.secret_data)
+
+  projects_ids = {
+    data_ingestion   = module.base_projects.data_ingestion_project_id,
+    governance       = module.base_projects.data_governance_project_id,
+    non_confidential = module.base_projects.non_confidential_data_project_id,
+    confidential     = module.base_projects.confidential_data_project_id
+  }
+}
+
+module "base_projects" {
+  source = "../../test//setup/base-projects"
+
+  org_id          = var.org_id
+  folder_id       = var.folder_id
+  billing_account = var.billing_account
+  region          = local.location
+}
+
+module "iam_projects" {
+  source = "../../test//setup/iam-projects"
+
+  data_ingestion_project_id        = module.base_projects.data_ingestion_project_id
+  non_confidential_data_project_id = module.base_projects.non_confidential_data_project_id
+  data_governance_project_id       = module.base_projects.data_governance_project_id
+  confidential_data_project_id     = module.base_projects.confidential_data_project_id
+  service_account_email            = var.terraform_service_account
+}
+
+resource "time_sleep" "wait_60_seconds_projects" {
+  create_duration = "60s"
+
+  depends_on = [
+    module.iam_projects
+  ]
+}
+
+
+resource "google_project_iam_binding" "remove_owner_role" {
```

Comment:
I think this is an important action the user will take, so might want to highlight it in the `main` example, but maybe consider even the main module? maybe within the iam things. it helps with the SHA findings.

Reply:
Elo, when you say 

> so might want to highlight it in the `main` example

Do you mean to move the resource `google_project_iam_binding.remove_owner_role` from the `harness.tf` file to the `main.tf` file in the standalone folder? 

Or do you mean to add a note in the `README` file of the standalone example telling the user that removing the owner from the project is to prevent a SHA finding?

Reply:
We added a note in the PR #270.

Could we resolve this conversation?